sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -9,7 +9,7 @@
 
 using namespace Azure::Security::KeyVault::Certificates;
 
-Azure::Response<CertificateOperationProperties> CreateCertificateOperation::PollUntilDoneInternal(
+Azure::Response<KeyVaultCertificateWithPolicy> CreateCertificateOperation::PollUntilDoneInternal(
     std::chrono::milliseconds period,
     Azure::Core::Context& context)
```

Comment:
We need to check for context cancellation.  Either inside the while(true) loop from this function or within the Poll( context ) function.

Reply:
we never do that anywhere else , so i'm not sure why we would suddenly do it here ? ism't that looked at during the SendRequest method , in the http pipeline ?